### PR TITLE
fix: exception should come in correct way to user on configuration page and corrected the message

### PR DESCRIPTION
**Pull request by Himanshi-Mirosoft**
Original issue raised: 
[Error while setting advanced image processing on ['jpeg', 'jpg', 'png', 'tiff', 'bmp'] · Issue #1250 · Azure-Samples/chat-with-your-data-solution-accelerator (github.com)](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/issues/1250)

Describe the bug
While the documentation (https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/main/docs/advanced_image_processing.md ) states that Advanced image processing can be done for ['jpeg', 'jpg', 'png', 'tiff', 'bmp'] , however configuring in the admin app the advanced image processing i do get the following errors:
Traceback (most recent call last): File "/tmp/8dcb7fb38d2a08c/pages/04_Configuration.py", line 404, in ConfigHelper.save_config_as_active(current_config) File "/tmp/8dcb7fb38d2a08c/batch/utilities/helpers/config/config_helper.py", line 204, in save_config_as_active ConfigHelper.validate_config(config) File "/tmp/8dcb7fb38d2a08c/batch/utilities/helpers/config/config_helper.py", line 224, in validate_config raise Exception( Exception: Advanced image processing has been enabled for document type pdf, but only ['jpeg', 'jpg', 'png', 'tiff', 'bmp'] file types are supported.

Expected behavior
please see link : (https://github.com/Azure-Samples/chat-with-your-data-solutionaccelerator/blob/main/docs/advanced_image_processing.md )

Image
Steps to reproduce
just attempt to configure advanced image processing in the Admin App as showing on the attachment to this issue
## Purpose

* ...

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
Open CWYD admin url
Click on 'Configuration' tab
Go to 'Document processing configuration' section 
Check the checkboxes in 'Use advanced image processing' column
Click on 'Save Configuration' button.
image

```
```

## What to Check
Verify that the following are valid
image

* ...

### CODE REVIEW

File: code/backend/batch/utilities/helpers/config/config_helper.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ def validate_config(config: dict):
                 and unsupported_advanced_image_processing_file_type
             ):
                 raise Exception(
-                    f"Advanced image processing has been enabled for document type {document_type}, but only {ADVANCED_IMAGE_PROCESSING_FILE_TYPES} file types are supported."
+                    f"Advanced image processing has not been enabled for document type {document_type}, but only {ADVANCED_IMAGE_PROCESSING_FILE_TYPES} file types are supported."
```
Comment (adamdougal) [inline code/backend/batch/utilities/helpers/config/config_helper.py:225]: I think a small change will make this read even better:
```suggestion
                    f"Advanced image processing has not been enabled for document type {document_type}, as only {ADVANCED_IMAGE_PROCESSING_FILE_TYPES} file types are supported."
```

Comment (Himanshi-Mirosoft) [reply code/backend/batch/utilities/helpers/config/config_helper.py]: I added this change

**Review by adamdougal — APPROVED** [review]
LGTM!

**Comment by Himanshi-Mirosoft** [comment]
Exception message issue got fixed, but We found one issue while checking checkbox quickly getting page refreshed and for that issue, we need more time for refractor this meanwhile we can merge this issue fix.

**Review by Roopan-Microsoft — APPROVED** [review]

**Pull request merged**